src/structures/interfaces/TextBasedChannel.js

```diff
@@ -172,88 +172,14 @@ class TextBasedChannel {
+   * Sends a typing indicator in the channel.
+   * @returns {Promise<void>} Resolves upon the typing status being sent
    * @example
-   * // Start typing in a channel, or increase the typing count by one
-   * channel.startTyping();
-   * @example
-   * // Start typing in a channel with a typing count of five, or set it to five
-   * channel.startTyping(5);
-   */
-  startTyping(count) {
-    if (typeof count !== 'undefined' && count < 1) throw new RangeError('TYPING_COUNT');
-    if (this.client.user._typing.has(this.id)) {
-      const entry = this.client.user._typing.get(this.id);
-      entry.count = count ?? entry.count + 1;
-      return entry.promise;
-    }
-
-    const entry = {};
-    entry.promise = new Promise((resolve, reject) => {
-      const endpoint = this.client.api.channels[this.id].typing;
-      Object.assign(entry, {
-        count: count ?? 1,
-        interval: this.client.setInterval(() => {
-          endpoint.post().catch(error => {
-            this.client.clearInterval(entry.interval);
-            this.client.user._typing.delete(this.id);
-            reject(error);
-          });
-        }, 9000),
-        resolve,
-      });
-      endpoint.post().catch(error => {
-        this.client.clearInterval(entry.interval);
-        this.client.user._typing.delete(this.id);
-        reject(error);
-      });
-      this.client.user._typing.set(this.id, entry);
-    });
-    return entry.promise;
-  }
-
-  /**
-   * Stops the typing indicator in the channel.
-   * The indicator will only stop if this is called as many times as startTyping().
-   * <info>It can take a few seconds for the client user to stop typing.</info>
-   * @param {boolean} [force=false] Whether or not to reset the call count and force the indicator to stop
-   * @example
-   * // Reduce the typing count by one and stop typing if it reached 0
-   * channel.stopTyping();
-   * @example
-   * // Force typing to fully stop regardless of typing count
-   * channel.stopTyping(true);
-   */
-  stopTyping(force = false) {
-    if (this.client.user._typing.has(this.id)) {
-      const entry = this.client.user._typing.get(this.id);
-      entry.count--;
-      if (entry.count <= 0 || force) {
-        this.client.clearInterval(entry.interval);
-        this.client.user._typing.delete(this.id);
-        entry.resolve();
-      }
-    }
-  }
-
-  /**
-   * Whether or not the typing indicator is being shown in the channel
-   * @type {boolean}
-   * @readonly
-   */
-  get typing() {
-    return this.client.user._typing.has(this.id);
-  }
-
-  /**
-   * Number of times `startTyping` has been called
-   * @type {number}
-   * @readonly
+   * // Start typing in a channel
+   * channel.sendTyping();
    */
-  get typingCount() {
-    return this.client.user._typing.get(this.id)?.count ?? 0;
+  async sendTyping() {
+    await this.client.api.channels(this.id).typing.post();
   }
```

Comment:
This is missing the return. Also why is there async / await if the promise is returned?
```suggestion
  sendTyping() {
    return this.client.api.channels(this.id).typing.post();
  }
```

Reply:
It returns a `Buffer` instance, which may not be elegant.
![image](https://user-images.githubusercontent.com/24852502/125616187-4e86de84-288e-434f-a361-e4ce0311b441.png)

Reply:
Ohh, so the async / await without returning is to returns the Promise without the result of the post call? Nice!